osgi/de.cau.cs.kieler.spviz.osgi.generate/osgi.spviz

```diff
@@ -14,35 +14,66 @@ SPViz OSGiViz {
 		connect OSGi.ServiceComponent.Required via OSGi.Bundle in Services
 		connect OSGi.ServiceInterface.ProvidedBy via OSGi.Bundle in Services
 	}
-	BundleDependencies {
+	
+	RequireBundleDependencies {
+		show OSGi.Bundle
+		connect OSGi.Bundle.Dependency
+	}
+	
+	BundlePackageDependencies {
+		show OSGi.Bundle
+		show OSGi.Package
+		connect OSGi.Bundle.PackageExportDependency
+		connect OSGi.Bundle.PackageImportDependency
+		connect OSGi.Bundle.PackageDependency
+	}
+	
+	AllDependencies {
 		show OSGi.Bundle
 		connect OSGi.Bundle.Dependency
-//		connect OSGi.Bundle via OSGi.Bundle.PackageDependency
```

Comment:
This comment was left in this file intentionally to show what I would like to be able to write in this grammar in the future: automatically connecting bundles via their package dependencies, instead of your new manual solution in the extractor.